pmd-core/src/main/java/net/sourceforge/pmd/PMDConfiguration.java

```diff
@@ -210,6 +211,23 @@ public LanguageVersionDiscoverer getLanguageVersionDiscoverer() {
         return languageVersionDiscoverer;
     }
 
+    /**
+     * Get the LanguageVersion specified as input
+     *
+     * @return The LanguageVersion.
+     */
+    public LanguageVersion getInputLanguageVersion() {
+        return inputLanguageVersion;
+    }
+
+    /**
+     * Set the LanguageVersion specified as input
```

Comment:
Also here: specify that the input language version may be null (or not, and check it's not null)

Reply:
Agreed, added more to the comment